app/javascript/packages/document-capture/components/acuant-capture.tsx

```diff
@@ -433,6 +437,12 @@ function AcuantCapture(
         });
       }
 
+      if (nativeCameraABTestingEnabled) {
```

Comment:
Great point! I went with `if (shouldStartAcuantCapture && nativeCameraABTestEnabled)` since that accurately encapsulates what I'm trying to test, i.e., if the user would otherwise be shown the Acuant SDK, check to see if we want to skip it for our test. Sound good?